### PR TITLE
[ZEPL-1553] Resource ACL documentation 

### PR DESCRIPTION
**Pull request by sravan-s**
This is the alpha draft for resource ACL documentation

### CODE REVIEW

**Comment by sravan-s** [comment]
ping @rbpark @minahlee @AhyoungRyu

File: docs/guide/resource_mgmt.md [hunk 1]
```diff
@@ -52,3 +52,53 @@ Create new resource if you want a different container resource type and image ty
 After a resource is deleted, all containers for that resource will be shutdown, and any notebooks that have the deleted resource attached will be set to the default resource.
 If you're trying to remove **Default Resource**, the next available resource in the resource list will be set as the next default resource.
 
+## Permissions for resources
+
+Inside an organization, users share its resources. To control the usage of
+resources in an organization, you can make use of resource permissions.
+After the introduction of resources permissions users can attach resources
+to notebooks and run them. View only and edit permissions set while sharing
+notebooks can only be used to restrict editting of notebooks, they wont
```
Comment (AhyoungRyu) [inline docs/guide/resource_mgmt.md]: wont -> won't

File: docs/guide/resource_mgmt.md [hunk 2]
```diff
@@ -52,3 +52,53 @@ Create new resource if you want a different container resource type and image ty
 After a resource is deleted, all containers for that resource will be shutdown, and any notebooks that have the deleted resource attached will be set to the default resource.
 If you're trying to remove **Default Resource**, the next available resource in the resource list will be set as the next default resource.
 
+## Permissions for resources
+
+Inside an organization, users share its resources. To control the usage of
+resources in an organization, you can make use of resource permissions.
+After the introduction of resources permissions users can attach resources
```
Comment (AhyoungRyu) [inline docs/guide/resource_mgmt.md]: I couldn't get what it means exactly; "the introduction of resources permissions users" ?

File: docs/guide/resource_mgmt.md [hunk 3]
```diff
@@ -52,3 +52,53 @@ Create new resource if you want a different container resource type and image ty
...
+
+There are three different permissions which can be applied to users
+or user groups such as team groups(members/ managers) or space user groups
+(members/ collaborators / managers)
+
+* Allow Attach, Detach Resource on a notebook
+* Allow Start, Stop, Execute Resource
+* Allow Modify Resource Setting
+
+Please note that these permissions does not depend on each other or have
```
Comment (AhyoungRyu) [inline docs/guide/resource_mgmt.md]: does -> do ?

File: docs/guide/resource_mgmt.md [hunk 4]
```diff
@@ -52,3 +52,53 @@ Create new resource if you want a different container resource type and image ty
...
+precedence over each other. You will have to be explicit on permissions.
+A user who doesnt have attach permission still can have edit permission.
```
Comment (AhyoungRyu) [inline docs/guide/resource_mgmt.md]: doesnt -> doesn't

File: docs/guide/resource_mgmt.md [hunk 5]
```diff
@@ -52,3 +52,53 @@ Create new resource if you want a different container resource type and image ty
...
+doesnt give edit permission to managers of that space.
+
+### Create a resource permission
+
+* Initially, only resource admins can configure permissions to resources.
+Later, users with edit permission would be able to do this.
+
+<img src="../../img/resource-add-permission-button.png" class="image-box middle-img" />
+
+* To add a permissions, go to edit page of a resource and click
```
Comment (AhyoungRyu) [inline docs/guide/resource_mgmt.md]: a permissions -> permissions ?

File: docs/guide/resource_mgmt.md [hunk 6]
```diff
@@ -52,3 +52,53 @@ Create new resource if you want a different container resource type and image ty
...
+* Allow Attach, Detach Resource on a notebook
+* Allow Start, Stop, Execute Resource
+* Allow Modify Resource Setting
+
+Please note that these permissions does not depend on each other or have
+precedence over each other. You will have to be explicit on permissions.
+A user who doesnt have attach permission still can have edit permission.
+Also, giving permission to a specific user group doesn't give permission
+to a higher user group. Example - giving edit permission to space users
+doesnt give edit permission to managers of that space.
```
Comment (AhyoungRyu) [inline docs/guide/resource_mgmt.md]: doesnt -> doesn't

**Review by AhyoungRyu — APPROVED** [review]
Looks Good. Just left minor comments ;)

File: docs/guide/resource_mgmt.md (hunk 3 above)
Comment (sravan-s) [reply docs/guide/resource_mgmt.md]: oki. I am a bit weak on do / does business 🥇

File: docs/guide/resource_mgmt.md (hunk 2 above)
Comment (sravan-s) [reply docs/guide/resource_mgmt.md]: I meant after resources permissions were introduced- ... changed

File: docs/guide/resource_mgmt.md [hunk 7]
```diff
@@ -52,3 +52,53 @@ Create new resource if you want a different container resource type and image ty
 After a resource is deleted, all containers for that resource will be shutdown, and any notebooks that have the deleted resource attached will be set to the default resource.
 If you're trying to remove **Default Resource**, the next available resource in the resource list will be set as the next default resource.
 
+## Permissions for resources
+
+Inside an organization, users share its resources. To control the usage of
+resources in an organization, you can make use of resource permissions.
+After the introduction of resource permissions users can attach resources
```
Comment (AhyoungRyu) [inline docs/guide/resource_mgmt.md:59]: Oh now I understood. "After the introduction of resource permissions, users can" : adding a comma will be better to understand I guess :)

Comment (khalidhuseynov) [reply docs/guide/resource_mgmt.md:59]: this sentence wasn't clear enough for me too. seems like previously users couldn't attach or run notebooks

File: docs/guide/resource_mgmt.md [hunk 8]
```diff
@@ -52,3 +52,53 @@ Create new resource if you want a different container resource type and image ty
...
+to notebooks and run them. View only and edit permissions set while sharing
```
Comment (khalidhuseynov) [inline docs/guide/resource_mgmt.md]: do we need to have link here to `View only and edit permission` documentation or `sharing notebooks` docs?

File: docs/guide/resource_mgmt.md [hunk 9]
```diff
@@ -52,3 +52,53 @@ Create new resource if you want a different container resource type and image ty
...
+notebooks can only be used to restrict editing of notebooks, they won't
+restrict the execution of notebooks.
+
+There are three different permissions which can be applied to users
+or user groups such as team groups(members/ managers) or space user groups
```
Comment (khalidhuseynov) [inline docs/guide/resource_mgmt.md]: do we need to have hyperlink here as well to `teams` and `space groups`?

File: docs/guide/resource_mgmt.md [hunk 10]
```diff
@@ -52,3 +52,53 @@ Create new resource if you want a different container resource type and image ty
...
+(members/ collaborators / managers)
```
Comment (minahlee) [inline docs/guide/resource_mgmt.md]: Can we use white space consistently? either (members / collaborators / managers) or (members/collaborators/managers).
Same in above line.

**Review by minahlee — COMMENTED** [review]
Nice doc sravan! LgTm.

**Comment by sravan-s** [comment]
Will merge after release

**Review by minahlee — APPROVED** [review]

**Pull request merged**